frontend/src/components/problem/ProblemDisplay.tsx

```diff
@@ -362,11 +393,25 @@ function ProblemDisplay(props: ProblemDisplayParams) {
           />
           <TitleDescriptionSeparator />
           <StyledMarkdownEditor
+            key={refreshEditor}
             placeholder="Write a nice description"
             defaultValue={newProblem.description}
             onChange={(getNewValue) => handleDescriptionChange(getNewValue())}
             readOnly={!problemEditable}
           />
+          {
+            (newProblem.description.length === 0 || newProblem.description === '\\\n') ? (
+              <TextButtonLink
+                onClick={() => {
+                  // Add default description and force refresh editor.
+                  newProblem.description = defaultDescription;
```

Comment:
nit: Change this to use `handleDescriptionChange` to avoid directly mutating state variables (also not sure where the refresh editor comes into play but I feel like doing this change might solve that issue)

Reply:
Done! ✅ 

That change alone doesn't fix the refresh editor issue, so I kept in the previous solution of increasing the key as well.